docs/testing-guidelines/WritingPesterTests.md

```diff
@@ -1,41 +1,41 @@
 ### Writing Pester Tests
+
 Note that this document does not replace the documents found in the [Pester](https://github.com/pester/pester "Pester") project.
 This is just some quick tips and suggestions for creating Pester tests for this project.
 The Pester community is vibrant and active, if you have questions about Pester or creating tests, the [Pester Wiki](https://github.com/pester/pester/wiki) has a lot of great information.
 As of January 2018, PowerShell Core is using Pester version 4 which has some changes from earlier versions.
 See [Migrating from Pester 3 to Pester 4](https://github.com/pester/Pester/wiki/Migrating-from-Pester-3-to-Pester-4) for more information.
 
-
-
 When creating tests, keep the following in mind:
 * Tests should not be overly complicated and test too many things
     * boil down your tests to their essence, test only what you need
 * Tests should be as simple as they can
 * Tests should generally not rely on any other test
 
 Examples:
-Here's the simplest of tests
+
+Here's the simplest of tests:
 
 ```powershell
 Describe "A variable can be assigned and retrieved" {
-    It "Create a variable and make sure its value is correct" {
+    It "Creates a variable and makes sure its value is correct" {
        $a = 1
-       $a | Should Be 1
+       $a | Should -Be 1
    }
 }
 ```
 
-If you need to do type checking, that can be done as well
+If you need to do type checking, that can be done as well:
 
 ```powershell
 Describe "One is really one" {
     It "Compare 1 to 1" {
        $a = 1
-       $a | Should Be 1
+       $a | Should -Be 1
     }
     It "1 is really an int" {
        $i = 1
-       $i.GetType() | Should Be "int"
+       $i.GetType() | Should -BeExactly "int"
```

Comment:
this should really be:
`$i | Should -BeOfType [int]`
and consistent with the next change

Reply:
Doing this then makes this example a duplicate of the one that follows it. Should we just remove this example instead of we don't want to recommend checking types this way?